File: rgb.py
from PIL import Image
import numpy as np
# Open the image
img = Image.open("/home/rguktrkvaleey/dhana.jpeg")
# Convert image to a NumPy array
img_array = np.array(img)

# Reshape the array to access individual pixel values
pixel_values = img_array.reshape(-1, 3)

# Write pixel values to CSV
np.savetxt("rgb_value.csv", pixel_values, delimiter=",", fmt="%d")

File: rgb.m
# Read the image
[img, map] = imread("/home/rguktrkvaleey/dhana.jpeg");

# Extract RGB channels
red = img(:,:,1);
green = img(:,:,2);
blue = img(:,:,3);

# Combine RGB channels into a matrix
rgb_values = [red(:), green(:), blue(:)];

# Save RGB values to CSV
csvwrite("rgb_values.csv", rgb_values);